### ISSUE TITLE
[BUG] Certain Youtube Videos Won't load

### ISSUE THREAD

**Issue by DraconicNEO** [yuliskov/SmartTube#534]
Hello There are some videos on Youtube that won't load at all in SmartTubeNext. Usually these videos don't Show up in search and when I do click on them They Just Give a toast saying `Playback Error Occurred 1` and afterwards it says video unavailable. It Usually happens with YouTube Movies, but it's also happened with a few Songs as well. Hopefully this issue can be resolved soon because it impacts the usability of SmartTubeNext as a YouTube client for my Fire TV.

**Comment by pirsir76** [comment]
Yes, I have this problem as well. Probably, everyone else has this problem as well.
All's I can say is that cross your fingers and it will be fixed somewhere down the track.
It appears this is not a github project that gets a lot of attention...

**Comment by DraconicNEO** [comment]
> 
> 
> Yes, I have this problem as well. Probably, everyone else has this problem as well.
> All's I can say is that cross your fingers and it will be fixed somewhere down the track.
> It appears this is not a github project that gets a lot of attention...

It's a shame really since there really isn't another custom client for YouTube TV.

**Comment by kwest12** [comment]
Yep, I'm getting something like this too.  When I try to play the unlisted videos that I've uploaded, I get "Video unavailable" and then a youtube link after it.  For example, "Video unavailable <a href="http://www.youtube.com/watch?v=aAGagjshrRW&feature=emb_err_woyt" target="_blank">

**Comment by beachbouy** [comment]
Issue is still ongoing. I get error 500 when trying to watch videos after only installing the app about an hour ago. It worked ok for the first few videos, but I think Youtube must have banned my SmartTube app for blocking the ads.
